File: library/src/main/java/com/example/library/dto/response/CategoryResponse.java
package com.example.library.dto.response;

import java.time.LocalDateTime;

import com.example.library.entity.Account;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CategoryResponse {

    private Integer id;

    private String categoryCode;

    private String categoryName;

    private Integer accountId;
        
    private LocalDateTime updatedAt;
    
    private LocalDateTime deleteAt;
    
    private Long amountBook;

}

File: MyApplication/app/src/main/java/com/example/myapplication/data/remote/dto/response/CategoryResponse.kt
package com.example.myapplication.data.remote.dto.response

import com.example.myapplication.domain.model.Account
import java.time.LocalDateTime

//package com.example.myapplication.data.remote.dto.response
//
//import com.example.myapplication.domain.model.Account
//import java.time.LocalDateTime

data class CategoryResponse(
    val id: Int,
    val categoryCode: String,
    val categoryName: String,
    val accountId: Int,
    val amountBook: Long,
    val updatedAt: LocalDateTime,
    val deleteAt: LocalDateTime?
)